bee-common/bee-common/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "bee-common"
+version = "1.0.0-alpha"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "Common utilities used across the bee framework"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = ["iota", "tangle", "bee", "framework", "common"]
+homepage = "https://www.iota.org"
+
+[features]
+alloc = []
+io = []
+
+[dev-dependencies]
+bee-common = { path = ".", features = ["alloc", "io"] }
```

Comment:
Is that actually necessary ?

Reply:
the tests require it :/

Reply:
Ah maybe that's because of the features ? Because the crates is automatically added for tests of the same crate